src/wolfgame.py

```diff
@@ -6994,13 +7004,16 @@ def vote_gamemode(var, wrapper, gamemode, doreply):
         return
 
     if gamemode not in var.GAME_MODES.keys():
-        match, _ = complete_match(gamemode, var.GAME_MODES.keys() - ["roles", "villagergame"] - var.DISABLED_GAMEMODES)
-        if not match:
-            if doreply:
-                wrapper.pm(messages["invalid_mode_no_list"].format(gamemode))
+        matches = complete_match(gamemode, var.GAME_MODES.keys() - ["roles", "villagergame"] - var.DISABLED_GAMEMODES)
```

Comment:
Style nit: change `["roles", "villagergame"]` for `{"roles", "villagergame"}` - i.e. curly braces instead of square brackets. Also update the lover one above.